### PR TITLE
shipshape-app#190: Make typographic scale fluid

### PR DESCRIPTION
**Pull request by danielmatthew**
Fixes https://github.com/talis/shipshape-app/issues/190

### CODE REVIEW

File: assets/stylesheets/bootstrap/_type.scss [hunk 1]
```diff
@@ -3,6 +3,19 @@
 // --------------------------------------------------
 
 
+@function em($pixels, $context: $browser-context) {
```
Comment (pads) [inline assets/stylesheets/bootstrap/_type.scss]: I think functions warrant doc blocks here. It's not clear how text scaling is applied. I used to remember this myself but I've forgotten!

Comment (pads) [inline assets/stylesheets/bootstrap/_type.scss]: And by explanation, I mean things like why ems are used and where pixels come into it.

File: assets/stylesheets/bootstrap/_variables.scss [hunk 2]
```diff
@@ -59,6 +59,8 @@ $link-hover-decoration: none !default;
 //
 //## Font, line-height, and color for body text, headings, and more.
 
+$browser-context: 16;
```
Comment (pads) [inline assets/stylesheets/bootstrap/_variables.scss]: If I remember correctly, this is like a 16px baseline?

File: assets/stylesheets/bootstrap/_variables.scss [hunk 3]
```diff
@@ -78,6 +78,9 @@ $font-size-h4:            ceil(($font-size-base * 1.538461538)) !default; // ~20
 $font-size-h5:            $font-size-base !default;
 $font-size-h6:            $font-size-base !default;
 
+$font-size-scale:         0.85;
```
Comment (junglebarry) [inline assets/stylesheets/bootstrap/_variables.scss]: Best check the effect on the iOS textfield focus/zoom issue.

Comment (danielmatthew) [inline assets/stylesheets/bootstrap/_variables.scss]: Gnnh.

Comment (danielmatthew) [inline assets/stylesheets/bootstrap/_variables.scss]: Although - as covered in Hipchat - this actually has no effect on our textboxes. The additions are only used to the heading scale, not body text.

**Pull request merged**